CapitalForecasting/Modules/hierarchical_modeling.R

```diff
@@ -23,6 +25,7 @@ hierarchical_modeling <- function(data, train_split_date) {
   data %>% 
     filter(year(date) <= train_split_date) %>% 
     model(arima = ARIMA(dollars)) %>% 
+    # asset forecast minus liabilities forecast equals working capital forecast
     reconcile(mint_arima = min_trace(arima, method = "mint_shrink")) %>% 
     refit(data)
 }
```

Comment:
At the end you should use return() and/or save() so that you can call the results